tests/modules/programs/zed-editor/extensions.nix

```diff
@@ -11,20 +11,50 @@
     ];
   };
 
+  home.homeDirectory = lib.mkForce "/build/hm-user";
```

Comment:
It looks like MacOS doesn't use `/build/` as the directory where builds are performed. I only have Linux machines. Do you have any idea on what's the correct value here for Mac so I can fix the test?

Reply:
I have a macbook. I can take a look at this later, when I get time.

Reply:
Sorry, i forgot about this...